src/components/List/List.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styles from './List.module.scss';
+import { Book, PageData } from '../../types/main';
+import { apiService } from '../../api/apiService';
+import Card from './Card/Card';
+import Pagination from './Pagination/Pagination';
+
+interface ListProps {
+  searchText: string;
+}
+
+interface ListState {
+  isLoading: boolean;
+  error: string;
+  pageNumber: number;
+  books: Book[];
+  pageData: PageData | null;
+}
+
+class List extends React.Component<ListProps, ListState> {
+  state = {
+    isLoading: true,
+    error: '',
+    pageNumber: 0,
+    books: [],
+    pageData: null,
+  };
+  componentDidMount(): void {

```

Comment:
```suggestion
  };

  componentDidMount(): void {
```
Use prettier before the commit.